website/templates/project/project_header.mako

```diff
@@ -40,6 +40,11 @@
                                 Files
                             </a>
                         </li>
+                        <li id="projectNavMetadata">
+                            <a href="${node['url']}metadata/" class="subnav-header">
+                                Metadata
+                            </a>
+                        </li>
```

Comment:
This block should be just above the files block. Otherwise it looks great.

Reply:
Thank you Brian, this has now been updated.